exercises/binary-search/.meta/solutions/binary_search.rb

```diff
@@ -2,15 +2,15 @@ class BinarySearch
   attr_reader :list
 
   def initialize(data)
-    fail ArgumentError unless data.sort == data
```

Comment:
I think this is meant to be here, since the entire idea of binary search doesn't work if the data is not sorted.

Reply:
Further inspection of the problem-specifications this should be gone... so this is a good change.